pkg/install/install.go

```diff
@@ -350,6 +356,11 @@ func writeCNIConfig(c config) {
 	netconf = strings.Replace(netconf, "__ETCD_ENDPOINTS__", getEnv("ETCD_ENDPOINTS", ""), -1)
 	netconf = strings.Replace(netconf, "__ETCD_DISCOVERY_SRV__", getEnv("ETCD_DISCOVERY_SRV", ""), -1)
 
+	err = isValidJSON(netconf)
```

Comment:
IMHO it makes sense to directly test this function without the `ginkgo` construct since we can't pass an invalid CNI config to the install method.